packages/app-generic-asset/src/asset-util.ts

```diff
@@ -1,10 +1,6 @@
-import { assetRegistry } from '@cennznet/crml-generic-asset';
-
-const assets = assetRegistry.reserveAssets();
-export const reservedAssets = assets.reduce((acc: { [index: string]: string }, asset) => {
-  const key = asset.id;
-  if (!asset.symbol.endsWith('-T')) {
-    acc[key] = asset.symbol;
-  }
-  return acc;
-}, {});
+export const reservedAssets = {
+  1: 'CENNZ',
+  2: 'CPAY',
+  16000: 'CENNZ-T',
```

Comment:
Would prefer we don't show `-T` tokens on MainNet. If it's easiest let's just remove them for now. Manually adding them for testing via UI is not too hard

Reply:
Gotcha